### PR TITLE
Add UseCallbackMemo example

### PR DESCRIPTION
**Pull request by pabloferro**
## Summary

Add UseCallbackMemo example

## Screenshots
Captura de Pantalla 2020-10-05 a la(s) 18 52 04

### CODE REVIEW

File: src/app/screens/Dashboard/screens/useCallbackMemo/index.tsx [hunk 1]
```diff
@@ -1,11 +1,49 @@
...
+  const [timer, setTimer] = useState(0);
+  useEffect(() => {
+    const interval = setInterval(() => setTimer(prevTimer => prevTimer + 1), INTERVAL);
+    return () => clearInterval(interval);
+  });
+
+  const averageRating = () => getAverageRating(user.movieHistory);
+  const saveProfile = () => GhibliService.saveProfile(user);
+
+  // TODO: Implement save on unmount
```
Comment (alejobh) [inline src/app/screens/Dashboard/screens/useCallbackMemo/index.tsx:35]: is this intented to live coding?

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@types/lodash": "^4.14.161",
```
Comment (alejobh) [inline package.json]: you are not using this

File: src/app/screens/Dashboard/screens/useCallbackMemo/index.tsx (hunk 1 above)
Comment (pabloferro) [reply src/app/screens/Dashboard/screens/useCallbackMemo/index.tsx:35]: Yes

File: package.json (hunk 2 above)
Comment (pabloferro) [reply package.json]: Noo, I'll remove it.

**Review by alejobh — APPROVED** [review]

**Pull request merged**